### ISSUE TITLE
Training for TinyYOLOv3

### ISSUE THREAD

**Issue by IlkayW** [OlafenwaMoses/ImageAI#679]
Hi,
will there be a setModelTypeAsTinyYOLOv3() option for DetectionModelTrainer any time soon?

**Comment by ekesdf** [comment]
@IlkayW 
I don't think so, bc tiny is not implemented. I mean it is possible to implement the other model, but it would acquire a huge effort so I think it will happen in the next month even if he would be working on it right now what i doubt
